agent/api/task/task.go

```diff
@@ -2338,8 +2338,21 @@ func (task *Task) dockerLinks(container *apicontainer.Container, dockerContainer
 }
 
 var getHostPortRange = utils.GetHostPortRange
+var getHostPort = utils.GetHostPort
 
+// dockerPortMap creates a port binding map for
+// (1) Ingress listeners for the service connect AppNet container in the service connect enabled bridge network mode task.
+// (2) Port mapping definied by customers in the task definition.
```

Comment:
nit: defined

Reply:
disregard, it's been addressed in part two :)

Reply:
Thanks for catching this. It will be fixed in Part 2 when it's going to be rebased against the feature branch!